src/trajectory.jl

```diff
@@ -185,8 +193,9 @@ function transition(
     z::PhasePoint
 ) where {T<:Real}
     # Create the corresponding static τ
-    n_steps = max(1, floor(Int, τ.λ / τ.integrator.ϵ))
-    static_τ = StaticTrajectory(τ.integrator, n_steps)
+    n_steps = max(1, floor(Int, τ.λ / nom_step_size(τ.integrator)))
+    integrator = jitter(rng, τ.integrator)
```

Comment:
This `jitter` call is duplicate. As the later `transition` call already jitters the step size.